arrocke/gloss-translation
# chore: Remove vagrant support

## What has changed


Remove support for Vagrant

## Connected Issues


closes #284 

## QA Steps


1. Review file changes in this PR.
2. Compare with file changes in chore: add vagrant support(#225) 
3. Ensure that what was added in previous PR is reverse in current PR.
 
## Post-Deployment



- [x ] Nothing required
